modules/op-release-notes-1-8.adoc

```diff
@@ -12,13 +12,198 @@ With this update, {pipelines-title} General Availability (GA) 1.8 is available o
+// Divyanshu Agrawal @divyansh42
+
+* With this update, you can use the `spec.status` value, `Cancelled`, instead of the deprecated `PipelineRunCancelled` status.
+// link:https://github.com/tektoncd/cli/pull/1554[(#1554)]
+// Vinamra Jain @vinamra28
+
+[id="operator-new-features-1-8_{context}"]
+=== Operator
+
+// [Hub]
+* With this update, as an administrator, you can configure your local {tekton-hub} instance to use a custom database rather than the default database.
+// (link:https://issues.redhat.com/browse/SRVKP-2309[SRVKP-2309 Pipeline as code: validate repository URL uniqueness])
+// Puneet Punamiya
+
+* With this update, as a cluster administrator, if you enable your local {tekton-hub} instance, it periodically refreshes the database so that changes in the catalog appear in the {tekton-hub} web console. You can adjust the period between refreshes.
++
+Previously, to add the tasks and pipelines in the catalog to the database, you performed that task manually or set up a cron job to do it for you.
+// (link:https://issues.redhat.com/browse/SRVKP-2252[SRVKP-2252 Refresh catalogs on schedule])
+// Puneet Punamiya
+
+* With this update, you can install and run a {tekton-hub} instance with minimal configuration. This way, you can start working with your teams to decide which additional customizations they might want.
+// (link:https://issues.redhat.com/browse/SRVKP-2250[SRVKP-2250 Install Hub with default configs])
+// Shiv Verma
+
+// [Chains]
+* Previously, the `chains-config` config map for {tekton-chains} in the `openshift-pipelines` namespace was automatically reset to default after upgrading the {pipelines-title} Operator. The current update fixes this issue. Now, you can configure key-value pairs in the `chains-config` config map, and those values persist after upgrading the {pipelines-title} Operator.
+// https://issues.redhat.com/browse/SRVKP-2349
+// Piyush Garg @piyush-garg
+
+* This update adds `GIT_SSL_CAINFO` to the `git-clone` task so you can clone secured repositories.
+// https://issues.redhat.com/browse/SRVKP-1423
+// @savitaashture
+
+[id="hub-new-features-1-8_{context}"]
+=== Hub
+
+* With this update, you can disable the login and rating features of your {tekton-hub} web console. To do so, omit adding a value for the `AUTH_BASE_URL` field in the `<tekton_hub_repository>/config/02-api/20-api-secret.yaml` file.
+// https://github.com/tektoncd/hub/pull/542
+// Shiv Pratap Verma @pratap0007
+
+* With this update, you don't need to log in to the {tekton-hub} web console to add resources from the catalog to the database. Now, these resources are automatically added when the catalog refresh API runs.
+// https://github.com/tektoncd/hub/pull/510
+// Shiv Pratap Verma @pratap0007
+
+* This update automatically refreshes the catalog every 30 minutes by calling the catalog refresh API job. This interval is user-configurable.
+// https://github.com/tektoncd/hub/pull/547
+// Shiv Pratap Verma @pratap0007
```

Comment:
Hi Shiv (@pratap0007). Please review and approve these release notes. Thanks!

Reply:
looks good

Reply:
@rolfedh lgtm